### ISSUE TITLE
check out this one

### ISSUE THREAD

**Issue by gijscor** [sarea/HelpMeOut#1]
This is also a similar company:

https://www.taskrabbit.com/

**Comment by gijscor** [comment]
but it's just in the US

**Comment by AliAtrash** [comment]
Agree, although this is business oriented. And so we found a place to copycat a thing or two xD But hey if the idea isn't concrete we can immediately re-shuffle.
